[ci] disable onnx py3 gcc5 build

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#31100 [ci] disable onnx py3 gcc5 build**

This appears to not work right now. Disabling pending an investigation.

Differential Revision: [D18928777](https://our.internmc.facebook.com/intern/diff/D18928777)

suo committed:
[ci] disable onnx py3 gcc5 build

This appears to not work right now. Disabling pending an investigation.

wanchaol reviewed (approved):
lint seems complaining.

suo committed:
Update on "[ci] disable onnx py3 gcc5 build"


This appears to not work right now. Disabling pending an investigation.

Differential Revision: [D18928777](https://our.internmc.facebook.com/intern/diff/D18928777)

suo committed:
Update on "[ci] disable onnx py3 gcc5 build"


This appears to not work right now. Disabling pending an investigation.

Differential Revision: [D18928777](https://our.internmc.facebook.com/intern/diff/D18928777)
